KsApi/mutations/inputs/CreateBackingInput.swift

```diff
@@ -4,31 +4,32 @@ public struct CreateBackingInput: GraphMutationInput {
   let amount: String
   let locationId: String?
   let paymentSourceId: String
-  let paymentType: String
   let projectId: String
   let rewardId: String?
+  let refParam: String?
 
   public init(
     amount: String,
     locationId: String?,
     paymentSourceId: String,
-    paymentType: String,
     projectId: String,
-    rewardId: String?
+    rewardId: String?,
+    refParam: String?
   ) {
     self.amount = amount
     self.locationId = locationId
     self.paymentSourceId = paymentSourceId
-    self.paymentType = paymentType
     self.projectId = projectId
     self.rewardId = rewardId
+    self.refParam = refParam
   }
 
   public func toInputDictionary() -> [String: Any] {
     var inputDictionary = [
       "amount": amount,
       "paymentSourceId": paymentSourceId,
-      "paymentType": paymentType,
+      // swiftlint:disable:next line_length
+      "paymentType": "credit_card", // this is temporary and will be removed once the mutation has been updated
```

Comment:
I'm guessing we can flag the `PaymentType` `enum` for removal too?

Reply:
Oh yeah good call.